### DISCUSSION TITLE
How to change portal language?

### DISCUSSION THREAD

**Discussion by thehijacker** [authelia/authelia#6516] [Q&A] [answered]
I have done Slovenian translation few months ago. Today session expired and saw the login page again, but it was in English. Slovenian translation is there in the code (sl_SI) but it doesn't open. Google says it should use browser language and my navigator.languages correctly shows

```
navigator.languages
['sl-SI', 'sl', 'en-GB', 'en']
```

But page is always in English. Appending _?lang=sl-SI_ to URL also doesn't help. Is this a bug perhaps? Using **authelia/authelia** docker image.

**Comment by james-d-elliott** [comment, marked as answer]
Authelia automatically respects the browser language. There is no translation for sl-SI for 4.37.5: https://github.com/authelia/authelia/tree/v4.37.5/internal/server/locales
